src/presence/__tests__/heartbeat-test.js

```diff
@@ -55,7 +55,7 @@ class Lolex {
 let lolex: Lolex;
 
 // type alias for Jest callback functions of type (boolean) => void
-type CallbackType = JestMockFn<$ReadOnlyArray<boolean>, void>;
+type CallbackType = JestMockFn<$ReadOnlyArray<void>, void>;
```

Comment:
This comment is out of date :slightly_smiling_face: 

Simplest fix is to update it in tandem.

But I think this also makes an excellent example of why it's good to avoid repeating details of the code in the comments. Here, the information this is really trying to add is something like "those type arguments are the types of the mock function's parameters and return." One way to say that would be
```
// Type parameters are: JestMockFn<TArguments: $ReadOnlyArray<*>, TReturn>
```

Reply:
Unfortunately `*` is [deprecated](https://flow.org/en/docs/linting/rule-reference/#toc-deprecated-type). In particular I'm not sure that wouldn't have the effect of `ReadOnlyArray<any>`, which – while accurate for Jest – is probably not what I want here.